ide/idea-iml-file.xml

```diff
@@ -30,6 +30,7 @@
             <sourceFolder url="file://$MODULE_DIR$/tools/stress/test/unit" isTestSource="true" />
             <sourceFolder url="file://$MODULE_DIR$/tools/fqltool/src" isTestSource="false" />
             <sourceFolder url="file://$MODULE_DIR$/tools/fqltool/test/unit" isTestSource="true" />
+            <sourceFolder url="file://$MODULE_DIR$/test/anttasks" isTestSource="true" />
```

Comment:
nit: there's a similar line to add in the netbeans file.

Reply:
@michaelsembwever you know what ... I will remove this from IDEA. If we plan to extract it all from `test`, this seems like unnecessary exercise.